core/test/debt-test.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import { causeDebt } from '../helpers/causeDebt';
+import getContract from '../src/contracts';
+import BigNumber from '../src/bignumber';
+import reset from './helpers/resetBlockchainFork';
+import { HARDHAT_PRIVATE_KEY, NETWORK } from '../helpers/constants';
+const HARDHAT_FORK_BLOCK_NUMBER = 14052140;
+
+const REMOTE_RPC_URL = process.env.REMOTE_RPC_URL || 'http://localhost:8545';
+
+describe('Debt Auction', () => {
+    beforeEach(async () => {
+        const provider = await reset(REMOTE_RPC_URL, HARDHAT_FORK_BLOCK_NUMBER, HARDHAT_PRIVATE_KEY, NETWORK);
```

Comment:
Why does this function need all those parameters if they can be directly imported from the `../helpers/constants`?

Reply:
since it's a test helper, i would not expect all the tests that ever reset anything to work with the same parameters. 

I can agree though that network and rpc url are most likely always the same and it makes sense to import them.